src/GraphNodeManagerViewExtension/GraphNodeManagerView.xaml

```diff
@@ -40,8 +40,8 @@
         <DockPanel Grid.Row="1">
             <Grid x:Name="FilterGrid" DockPanel.Dock="Top">
                 <ItemsControl x:Name="FilterItemControl"
-                              Margin="5"
-                              ItemsSource="{Binding FilterItems, UpdateSourceTrigger=PropertyChanged}">
+                                  Margin="5"
```

Comment:
Remove additional spacing to align correctly.

Reply:
This has been replaced by a custom user control FilterItemControl.xaml - do you still see misalignment @reddyashish ?